src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 2;
```

Comment:
write your variables like this
const LONG_TERM = 7;
const LONG_TERM_DISCOUNT = 50;

checklist[2]